.github/workflows/python-package.yml

```diff
@@ -11,30 +11,17 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```

Comment:
Why remove the matrix? I think it's important to run these test across all supported versions.

Reply:
Actually thats not entirely true. `pytest` should be run for all supported versions. I should probably be moved to another job.

Perhaps it would alos be best to have the `inn` intergration tests not run along with other `pytests` it should be a separate job too.

Reply:
My gut says we should pytest Ubuntu on 3.9, 3.11, and 1.13 and then 3.10 on Windows and 3.12 on macOS.  I will put together a PR for that.

Flake8 was written in Python so its results were sensitive to which Python it was running on.  Ruff is written in Rust and can run on a machine where Python is not even installed so its results are only based on the setting in `pyproject.toml`:
```toml
[tool.ruff]
target-version = "py39"
```